saleor/payment/tasks.py

```diff
@@ -18,7 +19,7 @@
 logger = logging.getLogger(__name__)
 
 
-def checkouts_with_funds_to_release():
+def transactions_to_release_funds():
     """Fetch checkouts that are ready release the funds.
```

Comment:
Would be also worth to update the description of the function. As after the changes it doesn't return checkouts but transactions